components/map.jsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import Mapsearch from './mapsearch.jsx';
+
+class Map extends React.Component {
+  constructor(props) {
+    super(props);
+    this.onSearch = this.onSearch.bind(this);
+  }
+
+  componentDidMount() {
+    const theMap = this.refs.map;
```

Comment:
Not sure if 'theMap' is a great variable name. I don't see it used anywhere else.
1. Do we need this one?
2. If we do, can we try something else like insertedMap? or mapConstant? IDK what it's really doing in the code.